scripting/get5/version.sp

```diff
@@ -1,4 +1,3 @@
-#tryinclude "manual_version.sp"
 #if !defined PLUGIN_VERSION
-#define PLUGIN_VERSION "0.8.0-dev"
+#define PLUGIN_VERSION "<VERSION>-<COMMIT_HASH>"
```

Comment:
I would suggest we use something else as the placeholder, as developers will get literally `<VERSION>-<COMMIT_HASH>` as their version number if they compile outside of CI.

Maybe:
`#define PLUGIN_VERSION "<DEVELOPMENT>"`

And then in the CI logic, replace `<DEVELOPMENT>` with `x.y.z-aabbccdd` in one go, instead of two calls to `sed`? Should be possible.

Reply:
Sounds good!

Reply:
Why the placeholder can't remain a meaningful "0.9.0-dev" or similar for someone checking this out & compiling it manually?

Reply:
Well, 0.9 as a fixed version forever might also be slightly odd?

Reply:
If you move the version back, you have the problem of not having a single source of truth for the version.

Reply:
How so? We can remove it from all the CI and use grep and see to grab for these values. I'm working on it right now and think I have a possible solution for this where the only version that exists will be in `version.sp` with that `tryinclude` still there until we can move away from Jenkins fully

Reply:
Also totally fine, but I was under the impression you wanted the version in the CI file. I think it's better in the `version.sp` file as well and then regex it out for tagging. My point was just that I think we should only have one single place where we change it.

Reply:
I think this approach is good.

Reply:
Depending on the amount of development and features going in, I think it would be nice to have something that is released fairly often, especially for testing new features for public use. We can only do so much testing as a group, and enabling people to just grab the latest and greatest and report bugs would be far better than catching them in a full release.